keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/SlackAppController.kt

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.slack.callbacks.ManualJudgmentCallbackHandler
+import com.slack.api.app_backend.interactive_components.response.ActionResponse
+import com.slack.api.bolt.App
+import com.slack.api.bolt.servlet.SlackAppServlet
+import org.springframework.stereotype.Component
+import javax.servlet.annotation.WebServlet
+
+@Component
+@WebServlet("/slack/notifications/callbacks")
```

Comment:
Just as a reminder, we'll need to update gate to match this URI, correct?

Reply:
already updated and deployed it!